{
  try {
    createRange(0,Integer.MAX_VALUE);
    fail("too large range accepted");
  }
 catch (  IllegalArgumentException e) {
    assertTrue("expected exception thrown",true);
  }
}
